From a6b9680284a4f62f62e38e52ce12276c849e373e Mon Sep 17 00:00:00 2001
From: ndifolcx <neilx.di.folco@intel.com>
Date: Fri, 23 Nov 2012 17:25:07 +0100
Subject: [PORT FROM MAIN] a1026.h alignment in kernel/bionic/external folders

BZ: 69837

a1026.h is duplicated in bionic/external headers directories.
Bionic version must be realigned on the kernel version.

Change-Id: Id2b1dc8988e0f81d608ffc675294f9700a331aba
Orig-Change-Id: I47c3d90bfa3d29ce7b53b50ee31b2e8daa9c03c5
Signed-off-by: ndifolcx <neilx.di.folco@intel.com>
Reviewed-on: http://android.intel.com:8080/75819
Reviewed-by: lab_cactus <lab_cactus@intel.com>
Reviewed-by: Barthes, FabienX <fabienx.barthes@intel.com>
Tested-by: Barthes, FabienX <fabienx.barthes@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 libc/kernel/common/linux/a1026.h |   51 +++----------------------------------
 1 files changed, 5 insertions(+), 46 deletions(-)

diff --git a/libc/kernel/common/linux/a1026.h b/libc/kernel/common/linux/a1026.h
index 1e5ce1b..8b04e5d 100644
--- a/libc/kernel/common/linux/a1026.h
+++ b/libc/kernel/common/linux/a1026.h
@@ -19,58 +19,17 @@
 #ifndef __LINUX_A1026_H
 #define __LINUX_A1026_H
 #include <linux/ioctl.h>
-#define A1026_MAX_FW_SIZE (32*1024)
+#include <linux/i2c.h>
 /* WARNING: DO NOT EDIT, AUTO-GENERATED CODE - SEE TOP FOR INSTRUCTIONS */
+#define A1026_MAX_FW_SIZE (128*1024)
 struct a1026img {
  unsigned char *buf;
  unsigned img_size;
-};
-/* WARNING: DO NOT EDIT, AUTO-GENERATED CODE - SEE TOP FOR INSTRUCTIONS */
-enum A1026_PathID {
- A1026_PATH_SUSPEND,
- A1026_PATH_INCALL_RECEIVER,
- A1026_PATH_INCALL_HEADSET,
-/* WARNING: DO NOT EDIT, AUTO-GENERATED CODE - SEE TOP FOR INSTRUCTIONS */
- A1026_PATH_INCALL_SPEAKER,
- A1026_PATH_INCALL_BT,
- A1026_PATH_VR_NO_NS_RECEIVER,
- A1026_PATH_VR_NO_NS_HEADSET,
-/* WARNING: DO NOT EDIT, AUTO-GENERATED CODE - SEE TOP FOR INSTRUCTIONS */
- A1026_PATH_VR_NO_NS_SPEAKER,
- A1026_PATH_VR_NO_NS_BT,
- A1026_PATH_VR_NS_RECEIVER,
- A1026_PATH_VR_NS_HEADSET,
-/* WARNING: DO NOT EDIT, AUTO-GENERATED CODE - SEE TOP FOR INSTRUCTIONS */
- A1026_PATH_VR_NS_SPEAKER,
- A1026_PATH_VR_NS_BT,
- A1026_PATH_RECORD_RECEIVER,
- A1026_PATH_RECORD_HEADSET,
 /* WARNING: DO NOT EDIT, AUTO-GENERATED CODE - SEE TOP FOR INSTRUCTIONS */
- A1026_PATH_RECORD_SPEAKER,
- A1026_PATH_RECORD_BT,
- A1026_PATH_CAMCORDER,
- A1026_PATH_INCALL_TTY
-/* WARNING: DO NOT EDIT, AUTO-GENERATED CODE - SEE TOP FOR INSTRUCTIONS */
-};
-enum A1026_NS_states {
- A1026_NS_STATE_AUTO,
- A1026_NS_STATE_OFF,
-/* WARNING: DO NOT EDIT, AUTO-GENERATED CODE - SEE TOP FOR INSTRUCTIONS */
- A1026_NS_STATE_CT,
- A1026_NS_STATE_FT,
- A1026_NS_NUM_STATES
 };
-/* WARNING: DO NOT EDIT, AUTO-GENERATED CODE - SEE TOP FOR INSTRUCTIONS */
 #define A1026_IOCTL_MAGIC 'u'
-#define A1026_BOOTUP_INIT _IOW(A1026_IOCTL_MAGIC, 0x01, struct a1026img *)
-#define A1026_SET_CONFIG _IOW(A1026_IOCTL_MAGIC, 0x02, enum A1026_PathID)
-#define A1026_SET_NS_STATE _IOW(A1026_IOCTL_MAGIC, 0x03, enum A1026_NS_states)
-/* WARNING: DO NOT EDIT, AUTO-GENERATED CODE - SEE TOP FOR INSTRUCTIONS */
-#define A1026_SET_MIC_ONOFF _IOW(A1026_IOCTL_MAGIC, 0x50, unsigned)
-#define A1026_SET_MICSEL_ONOFF _IOW(A1026_IOCTL_MAGIC, 0x51, unsigned)
-#define A1026_READ_DATA _IOR(A1026_IOCTL_MAGIC, 0x52, unsigned)
-#define A1026_WRITE_MSG _IOW(A1026_IOCTL_MAGIC, 0x53, unsigned)
+#define A1026_BOOTUP_INIT _IO(A1026_IOCTL_MAGIC, 0x01)
+#define A1026_SUSPEND _IO(A1026_IOCTL_MAGIC, 0x02)
 /* WARNING: DO NOT EDIT, AUTO-GENERATED CODE - SEE TOP FOR INSTRUCTIONS */
-#define A1026_SYNC_CMD _IO(A1026_IOCTL_MAGIC, 0x54)
-#define A1026_SET_CMD_FILE _IOW(A1026_IOCTL_MAGIC, 0x55, unsigned)
+#define A1026_ENABLE_CLOCK _IO(A1026_IOCTL_MAGIC, 0x03)
 #endif
-- 
1.7.4.1

